backend/FwLite/FwDataMiniLcmBridge/Api/RichTextMapping.cs

```diff
@@ -0,0 +1,486 @@
+        FwTextPropType.ktptHyphenate,
+        FwTextPropType.ktptWidowOrphanControl,
+        FwTextPropType.ktptMaxLines,
+        FwTextPropType.ktptCellBorderWidth,
+        FwTextPropType.ktptCellSpacing,
+        FwTextPropType.ktptCellPadding,
+        FwTextPropType.ktptSetRowDefaults,
+        FwTextPropType.ktptRelLineHeight,
+        FwTextPropType.ktptTableRule,
+    ];
+
+    public static readonly FwTextPropType[] StringProps = [
+        FwTextPropType.ktptCharStyle,
+        FwTextPropType.ktptFontFamily,
+        FwTextPropType.ktptNamedStyle,
+        FwTextPropType.ktptBulNumTxtBef,
+        FwTextPropType.ktptBulNumTxtAft,
+        FwTextPropType.ktptFontVariations,
+        FwTextPropType.ktptParaStyle,
+        FwTextPropType.ktptTabList,
+        FwTextPropType.ktptCustomBullet,
+        FwTextPropType.ktptBulNumFontInfo,
+        FwTextPropType.ktptWsStyle,
+        FwTextPropType.ktptFieldName,
+    ];
+
+    public delegate ref T MapPropTypeToRichSpan<T>(RichSpan span);
+
+    public static MapPropTypeToRichSpan<int?> MapToDelegateInt(FwTextPropType type)
+    {
+        return type switch
+        {
+            FwTextPropType.ktptFontSize => span => ref span.FontSize,
+            FwTextPropType.ktptOffset => span => ref span.Offset,
+            FwTextPropType.ktptFirstIndent => span => ref span.FirstIndent,
+            FwTextPropType.ktptLeadingIndent => span => ref span.LeadingIndent,
+            FwTextPropType.ktptTrailingIndent => span => ref span.TrailingIndent,
+            FwTextPropType.ktptSpaceBefore => span => ref span.SpaceBefore,
+            FwTextPropType.ktptSpaceAfter => span => ref span.SpaceAfter,
+            FwTextPropType.ktptTabDef => span => ref span.TabDef,
+            FwTextPropType.ktptLineHeight => span => ref span.LineHeight,
+            FwTextPropType.ktptRightToLeft => span => ref span.RightToLeft,
+            FwTextPropType.ktptPadLeading => span => ref span.PadLeading,
+            FwTextPropType.ktptPadTrailing => span => ref span.PadTrailing,
+            FwTextPropType.ktptMarginTop => span => ref span.MarginTop,
+            FwTextPropType.ktptPadTop => span => ref span.PadTop,
+            FwTextPropType.ktptPadBottom => span => ref span.PadBottom,
+            FwTextPropType.ktptBorderTop => span => ref span.BorderTop,
+            FwTextPropType.ktptBorderBottom => span => ref span.BorderBottom,
+            FwTextPropType.ktptBorderLeading => span => ref span.BorderLeading,
+            FwTextPropType.ktptBorderTrailing => span => ref span.BorderTrailing,
+            FwTextPropType.ktptBulNumScheme => span => ref span.BulNumScheme,
+            FwTextPropType.ktptBulNumStartAt => span => ref span.BulNumStartAt,
+            FwTextPropType.ktptDirectionDepth => span => ref span.DirectionDepth,
+            FwTextPropType.ktptKeepWithNext => span => ref span.KeepWithNext,
+            FwTextPropType.ktptKeepTogether => span => ref span.KeepTogether,
```

Comment:
I see there are a handful of properties that are handled as ints both in Lcm and your code, but they're marked as FwTextPropVar.ktpvEnum and in some cases they're commented as being "ComBools". So, most of them are conceptually nullable booleans. I think the rule is just 0 == false, 1 == true everything else could probably be viewed as null/unspecified.

Right now we're primarily just trying to round trip everything, but that might be something worth improving now or later by mapping them to bools or enums.

The ones I see are: ktptKeepTogether, ktptKeepWithNext, ktptRightToLeft, ktptWidowOrphanControl, ktptBulNumScheme, ktptHyphenate, ktptSetRowDefaults.

Reply:
Yeah that's a good point, I figure we'll do this until we want to actually use those props. The annoying thing is that a lot of these aren't even used anymore and I don't know which is which. All I know for sure is that NamedStyle, Ws, and ObjData are used heavily.